as-ap-injbeamdiag/as_ap_injbeamdiag/main.py

```diff
@@ -4,13 +4,41 @@
 import time as _time
 
 import epics as _epics
+import numpy as _np
+
 from siriuspy.callbacks import Callback as _Callback
-from siriuspy.devices import DeviceSet as _DeviceSet, SOFB as _SOFB
+from siriuspy.devices import (
+    DeviceSet as _DeviceSet,
+    SOFB as _SOFB,
+    DCCT as _DCCT,
+)
 
 from . import csdev as _csdev
 
 
-class App(_DeviceSet, _Callback):
+class SOFTDevices(_DeviceSet):
+    """."""
+
+    def __init__(self):
+        """Initialize the instance."""
+        props = ('RawReadings-Mon',)
+        self.dcct_bo = _DCCT(_DCCT.DEVICES.BO, props2init=props)
+        props = ('SPassOrbX-Mon', 'SPassOrbY-Mon', 'SPassSum-Mon')
+        # self.trajx_tb = _np.
+        self.sofb_tb = _SOFB(_SOFB.DEVICES.TB, props2init=props)
+        self.sofb_tb.pv_object('SPassSum-Mon').add_callback(
+            self._update_tb_spass
+        )
+        props = ('MTurnOrbX-Mon', 'MTurnOrbY-Mon', 'MTurnSum-Mon')
+        self.sofb_bo = _SOFB(_SOFB.DEVICES.BO, props2init=props)
+        devs = (self.sofb_tb, self.sofb_bo)
```

Comment:
dcct_bo missing in this list
```suggestion
        devs = (self.sofb_tb, self.sofb_bo, self.dcct_bo)
```

besides, is the auto_monitor on for all these PVs?